pkgs/development/compilers/dotnet/default.nix

```diff
@@ -70,6 +70,11 @@ let
       // {
         inherit callPackage fetchNupkg buildDotnetSdk;
 
+        generateBuildDotnetInputScript = writeScript "generate-build-dotnet-input.sh" (
```

Comment:
Seems good, however (I'm sorry for doing bikeshedding), I'd go with the name `dotnet-buildsdk-input` (in the same vein as the `nix-prefetch-*` packages).

Nothing specially against this name though, up to you 😄

Reply:
well, I wasn't a fan of that name either but didn't have any better ideas 😅 changed to your suggestion

Reply:
> I'm sorry for doing bikeshedding

I think naming is important, so I wouldn't consider this bikeshedding when the focus of the PR this exact thing.

IMO the name is still a bit confusing, because there's no verb.  It's meant to be used with `buildDotnetSdk`, so my instinct would have been to call it `generateDotnetSdk` (or `generate-dotnet-sdk`).  Maybe 'prefetch' would also be a good verb?  Neither are used in a ton of places from what I can tell.  There are also similar tools called '2nix' or '-to-nix'.

Also, as a tangent, I always struggle with naming conventions in nix, e.g. `generateDotnetSdk` vs. `generate-dotnet-sdk`.

Reply:
I just wanted to put in my 2p, but I wouldn't object to any of these.

Reply:
> `generate-dotnet-sdk-input` would be even clearer, but also the most verbose, and we'd be almost back at square one 😅 

I believe the `-input` is overly verbose and possibly redundant even. At least other nix code generators don't seem to add this from what I know.

I'm okay with either one though, I'll only be copy pasting the name around most likely 😅